app/src/main/java/com/voipgrid/vialer/CallActivity.java

```diff
@@ -427,13 +418,22 @@ private void toggleSpeaker() {
     }
 
     // Mute or un-mute a call when the user presses the button.
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
```

Comment:
I think we will have to use compat with background colour tints: https://stackoverflow.com/a/46110196